x-pack/plugin/ent-search/qa/rest/src/yamlRestTest/resources/rest-api-spec/test/entsearch/225_query_ruleset_list.yml

```diff
@@ -0,0 +1,161 @@
+setup:
+  - skip:
+      version: " - 8.8.99"
+      reason: Introduced in 8.9.0
+  - do:
+      query_ruleset.put:
+        ruleset_id: test-query-ruleset-1
+        body:
+          ruleset_id: test-query-ruleset-1
```

Comment:
Do we need to include the `ruleset_id` again as part of the body? It shouldn't be necessary with it being part of the path, right?

Reply:
You're right, this also leads to weirdness when validating.

Reply:
Related to this PR: https://github.com/elastic/elasticsearch/pull/96988#discussion_r1237540320